chapter04/ex_4_3.pl

```diff
@@ -0,0 +1,38 @@
+#!perl
+use warnings;
+use strict;
+
+my @fred = above_average(1..10);
+print "\@fred is @fred\n";
+print "(Should be 6 7 8 9 10)\n";
+my @barney = above_average(100, 1..10);
+print "\@barney is @barney\n";
+print "(Should be just 100)\n";
+
+sub average {
+  if (@_ == 0) { 
+  	return
+  }
+  my $count = @_;
```

Comment:
Might  be a good place  to use <code> scalar </code> since we covered it in previous chapters.